.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,40 @@
-**Is your pull request related to a problem? Please describe.**
+# Description
 
-A link to an issue or a clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+Please include a summary of the change and which issue is fixed. Please also
+include relevant motivation and context.
 
-**Describe the solution you have implemented**
-A clear and concise description of what happens.
+Fixes # (issue)
 
-**Additional context**
-Add any other context or screenshots about the pull request here.
\ No newline at end of file
+## Type of change
+
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not
+  work as expected)
+- This change requires an update of [cucumber.io/docs](https://cucumber.io/docs)
+
+## Note to contributors
+
+If your change may impact contributors, explain it here, and update README.md
+and CONTRIBUTING.md accordingly.
+
+## Update required of cucumber.io/docs
+
+If the [Cucumber documentation](https://cucumber.io/docs/) requires some update,
+submit a PR to [cucumber/docs](https://github.com/cucumber/docs/) and
+references it here.
+
+Refs cucumber/docs/pull/(pull-number)
+
+# Checklist:
+
+You can think of your PR to be ready for review once the following checklist is
+complete. You can also add some checks if you want to.
+
+- [ ] Tests to check the changes in that PR have been added
+- [ ] New and existing tests are passing locally and on the CI
+- [ ] `bundle exec rubocop` reports no offense
```

Comment:
Does rubocop not run as part of CI?

Reply:
Yes it is.
But it is also really frustrating to have the CI failing because of rubocop.

This is a way to warn the developer before it happened.